### ISSUE TITLE
Discussion: Chaining like python magicmock

### ISSUE THREAD

**Issue by 0xR** [tjwudi/magicmock.js#1]
When using the python MagicMock it is possible to chain methods to mock nested calls:

``` python
requests.get.return_value.json.return_value = {'hello': 'world'}

requests.get('url.com').json() // is {'hello': 'world'}
```

Is this something that will be supported in magicmock.js in the future? Would it even be possible?

**Comment by meetwudi** [comment]
Yes, I think this would be totally possible, and would be awesome. Allow me one to two days to investigate on this and give you feedback ;) Thanks!
